test/unit/server_spec.js

```diff
@@ -1,6 +1,6 @@
 const { expect } = require('chai')
 const sinon = require('sinon')
-const proxyquire = require('proxyquire')
+const proxyquire = require('proxyquire').noCallThru()
```

Comment:
Wondering what this does

Reply:
@tavakyan the `noCallThru()` addition prevents the original code from being invoked at all.  See https://github.com/thlorenz/proxyquire#prevent-call-thru-for-all-future-stubs-resolved-by-a-proxyquire-instance